CMakeLists.txt

```diff
@@ -193,8 +193,8 @@ add_definitions(-DLLVM_VERSION_PATCH=${LLVM_VERSION_PATCH})
 if(EMBED_USE_LLVM)
   include(embed_clang)
 else()
-  find_package(Clang REQUIRED)
-  include_directories(SYSTEM ${CLANG_INCLUDE_DIRS})
+  find_package(LibClang REQUIRED)
```

Comment:
Is this going to work with static builds? It's using e.g. [CLANG_EXPORTED_TARGETS](https://github.com/iovisor/bpftrace/blob/master/src/ast/CMakeLists.txt#L78) which is defined in `ClangConfig.cmake` somewhere in the system path.

Reply:
It looks like the Alpine build is failing on many undefined reference to Clang functions. This change is probably the reason.